### DISCUSSION TITLE
Where is the source code of the function ext_module.nms() in mmcv.ops.num.py?

### DISCUSSION THREAD

**Discussion by xiaolifeimianbao** [open-mmlab/mmcv#2434] [General]
I wanna modify the code of ext_module.nms to be adapted my experiment, but I can’t find it. So please take time out of your busy schedule to give an answer. Thank you.

**Comment by zhouzaida** [comment]
There are three files related to nms.

- https://github.com/open-mmlab/mmcv/blob/master/mmcv/ops/csrc/pytorch/nms.cpp
- https://github.com/open-mmlab/mmcv/blob/master/mmcv/ops/csrc/pytorch/cpu/nms.cpp
- https://github.com/open-mmlab/mmcv/blob/master/mmcv/ops/csrc/pytorch/cuda/nms_cuda.cu

**Comment by zhouzaida** [comment]
Hi, I converted this issue to dicussion because it is recommended to open a discussion for this kind of issue.
